# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/wasm/WasmCraneliftCompile.cpp
# Commit: b9d32ef0ad83
# Full Hash: b9d32ef0ad835c014d1ced7577ed3d4c47a00031
# Author: Lars T Hansen <lhansen@mozilla.com>
# Date: 2021-10-18 16:09:29
# Regressor Bug: 1727084
# File Overlap Count: 1
# Description:
#   Bug 1727084 - Memory64 - Bulk memory operations. r=yury
#   
#   Implement all bulk memory operations on 64-bit memories.  Only
#   out-of-line paths are supported for memory.copy and memory.fill, for
#   now.
# ==============================================================================

diff -r 359e414a1d7f -r b9d32ef0ad83 js/src/wasm/WasmCraneliftCompile.cpp
--- a/js/src/wasm/WasmCraneliftCompile.cpp	Mon Oct 18 07:31:54 2021 +0000
+++ b/js/src/wasm/WasmCraneliftCompile.cpp	Mon Oct 18 07:31:55 2021 +0000
@@ -47,21 +47,21 @@
     case BD_SymbolicAddress::RefFunc:
       return SymbolicAddress::RefFunc;
     case BD_SymbolicAddress::MemoryGrow:
-      return SymbolicAddress::MemoryGrow;
+      return SymbolicAddress::MemoryGrowM32;
     case BD_SymbolicAddress::MemorySize:
-      return SymbolicAddress::MemorySize;
+      return SymbolicAddress::MemorySizeM32;
     case BD_SymbolicAddress::MemoryCopy:
-      return SymbolicAddress::MemCopy32;
+      return SymbolicAddress::MemCopyM32;
     case BD_SymbolicAddress::MemoryCopyShared:
-      return SymbolicAddress::MemCopyShared32;
+      return SymbolicAddress::MemCopySharedM32;
     case BD_SymbolicAddress::DataDrop:
       return SymbolicAddress::DataDrop;
     case BD_SymbolicAddress::MemoryFill:
-      return SymbolicAddress::MemFill32;
+      return SymbolicAddress::MemFillM32;
     case BD_SymbolicAddress::MemoryFillShared:
-      return SymbolicAddress::MemFillShared32;
+      return SymbolicAddress::MemFillSharedM32;
     case BD_SymbolicAddress::MemoryInit:
-      return SymbolicAddress::MemInit32;
+      return SymbolicAddress::MemInitM32;
     case BD_SymbolicAddress::TableCopy:
       return SymbolicAddress::TableCopy;
     case BD_SymbolicAddress::ElemDrop: